### PR TITLE
Killing four bugs with one PR?

### LINKED ISSUE #123: TextAlignment doesn't work
### Bug Description
I'm using math:TextView for displaying math text and all works well, but when I try to display it with center alignment with the attribute TextAlignment="Center" it doesn't work. How can I fix it?

### PR DESCRIPTION
**Pull request by Happypig375**
This should finish #123, #136, #137 and possibly #138.

### CODE REVIEW

**Comment by Happypig375** [comment]
Now the view output of CSharpMath.Avalonia and CSharpMath.SkiaSharp are the same. 
image

Should we decrease the default font size (50)? Seems a bit too large.

File: CSharpMath.Forms.Example/CSharpMath.Forms.Example.UWP/CSharpMath.Forms.Example.UWP.csproj [hunk 1]
```diff
@@ -171,9 +171,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms" Version="4.3.0.908675" />
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.8</Version>
-    </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.9" />
```
Comment (FoggyFinder) [inline CSharpMath.Forms.Example/CSharpMath.Forms.Example.UWP/CSharpMath.Forms.Example.UWP.csproj:174]: why not `6.2.10` then?

Comment (Happypig375) [reply CSharpMath.Forms.Example/CSharpMath.Forms.Example.UWP/CSharpMath.Forms.Example.UWP.csproj:174]: It updated during install of Xamarin.Essentials. Will modify to 6.2.10.

**Comment by FoggyFinder** [comment]
Something is not right with these changes.
Text in display mode reacts on mouse dragging:

**Comment by Happypig375** [comment]
Maybe we should remove panning/dragging. It is from a time when view sizes was buggy and views are required to fill the entire available size. Now that view sizes is getting fixed (#32), we should replace all uses of panning/dragging with ScrollViews.

If we are to remove panning/dragging anyways, we might as well ignore this issue for now.

**Comment by FoggyFinder** [comment]
> we should replace all uses of panning/dragging with ScrollViews.

What about line breaks then ? (or I didn't get the point how it will be implemented)

**Comment by Happypig375** [comment]
Panning/dragging is independent of line breaking. People can already use TextVoew in ScrollView as seen in #136.

**Pull request merged**